src/custom/state/orders/hooks.ts

```diff
@@ -72,6 +72,19 @@ type UpdateLastCheckedBlockCallback = (updateLastCheckedBlockParams: UpdateLastC
 
 type GetOrderByIdCallback = (id: OrderID) => SerializedOrder | undefined
 
+type OrderTypeKeys = 'pending' | 'expired' | 'fulfilled' | 'cancelled'
+function _concatOrdersState(state: OrdersState[ChainId], keys: OrderTypeKeys[]) {
```

Comment:
If this function is only used once, why do we need to pass the `keys`? 

Also, the name is strange, right? 
It doesn't concat anything afait. It just `initiateOrderState`, probably add some coment saying what are u trying to accomplish. For example, initialise all the required keys from the order state. It's possible that a persisted order from a previous version might be missing some property.